<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormConfig</class>
 <widget class="QDialog" name="FormConfig">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="TButtonConfigDbSave">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/PNG/48x48/Filesaveas-48x48.png</normaloff>:/images/PNG/48x48/Filesaveas-48x48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="GBoxConfigDb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>481</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Configuração de Acesso a Banco</string>
   </property>
   <widget class="QLabel" name="ConfigLBDbHost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Endereço do Banco:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Porta:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbPort">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Informe a porta de conexão com o banco!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nome do Banco:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Informe o nome do banco!</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbUser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usuário:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbUser">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Informe o usuário de conexão com o banco!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbPass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Senha:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbPass">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Informe a senha de conexão com banco!</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonTestConn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Testar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/24x24/reload-24x24.png</normaloff>:/images/PNG/24x24/reload-24x24.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbTestResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reultado:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbTestResult">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>370</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Mostra mensagens de erro ou sucesso!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbHost">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Informe o endereço de conexão com o banco!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>ConfigLEDbName</zorder>
   <zorder>ConfigLBDbHost</zorder>
   <zorder>ConfigLBDbPort</zorder>
   <zorder>ConfigLEDbPort</zorder>
   <zorder>ConfigLBDbName</zorder>
   <zorder>ConfigLBDbUser</zorder>
   <zorder>ConfigLEDbUser</zorder>
   <zorder>ConfigLBDbPass</zorder>
   <zorder>ConfigLEDbPass</zorder>
   <zorder>TButtonTestConn</zorder>
   <zorder>ConfigLBDbTestResult</zorder>
   <zorder>ConfigLEDbTestResult</zorder>
   <zorder>ConfigLEDbHost</zorder>
  </widget>
  <widget class="QToolButton" name="TButtonConfigDbExit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/PNG/48x48/Exithome-48x48.png</normaloff>:/images/PNG/48x48/Exithome-48x48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
